coffee-chats/src/main/java/com/google/step/coffee/HttpError.java

```diff
@@ -0,0 +1,22 @@
+package com.google.step.coffee;
+
+import com.google.gson.annotations.Expose;
+
+public class HttpError extends Exception {
+  @Expose
+  private int errorCode;
```

Comment:
I don't know what style does java style guide says, but can you follow it on both these lines?

In one case `@Expose` is on separate line here, in another on the same.